terms.html

```diff
@@ -53,6 +59,8 @@
   <dd>
 A thing with distinct and independent existence, such as a person,
 organization, or device that performs one or more roles in the ecosystem.
+For accountability and other purposes, it can be relevant to know 
```

Comment:
Surprised to not see "entity" defined here as being `subject`, `issuer`, `holder` or `verifier`... or "other roles related to the vc data model to be defined in the future.

Reply:
Please make  a concrete proposal for changes if you think that would be required, so we can get this PR done.

Reply:
This commend can be resolved when you accept or dismiss suggestions from @brentzundel and @TallTed